lib/bots/versionbot.ts

```diff
@@ -84,12 +84,24 @@ interface FSError {
     message: string;
 }
 
+interface VersionBotConfiguration extends GithubBotConfiguration {
+    githubbot: {
+        versionbot: {
+            minimum_approvals?: number;
+            approved_reviewers?: string[];
+            maintainers?: string[];
+        };
+    };
+}
+
 // The VersionBot is built on top of GithubBot, which does all the heavy lifting and scheduling.
 // It is designed to check for valid `versionist` commit semantics and alter (or merge) a PR
 // accordingly.
 export class VersionBot extends GithubBot.GithubBot {
     // Flowdock Adapter
     private flowdock: FlowdockAdapter;
+    private mergeLabel = 'procbots/versionbot/ready-to-merge';
+    private ignoreLabel = 'procbots/versionbot/no-checks';
```

Comment:
Doesn't need to live in the class though. It's not something you want exposed, and it's not mutable state, so it could just be constants that live above. Let's you drop this `this.` everywhere too.

This is bikeshedding territory though. `readonly` or `const` is fine by me.